client/src/frontend/components/entity/useEntityHeadContent/index.js

```diff
@@ -0,0 +1,143 @@
+import { useFromStore } from "hooks";
+import { useTranslation } from "react-i18next";
+
+const appendHeadTitle = (entityTitle, headTitle) => {
+  if (!headTitle) return entityTitle;
+  return `${entityTitle} | ${headTitle}`;
+};
+
+const maybeAppendParentTitle = (entityTitle, parent) => {
+  const { socialTitle, titlePlaintext } = parent?.attributes ?? {};
+  const parentTitle = socialTitle ?? titlePlaintext;
+  if (parentTitle) return `${entityTitle} | ${parentTitle}`;
+  return entityTitle;
+};
+
+const TYPE_MAP = {
+  journalVolumes: "glossary.volume_one",
+  journalIssues: "glossary.issue_truncated_one"
+};
+
+const maybeSubstituteNumber = (titlePlaintext, number, type, t) => {
+  const typeStr = TYPE_MAP[type];
+  if (!number || !typeStr) return titlePlaintext;
+  return `${t(typeStr)} ${number}`;
+};
+
+const maybeSocialTitle = (socialTitle, ...args) => {
+  if (socialTitle) return socialTitle;
+  return maybeSubstituteNumber(...args);
+};
+
+const maybePrepend = (entityTitle, prepend) => {
+  if (!prepend) return entityTitle;
+  return `${prepend} | ${entityTitle}`;
+};
+
+export const useMetaTitle = (entity, parent, prepend) => {
+  const { t } = useTranslation();
+  const settings = useFromStore("settings", "select");
+
+  if (!entity?.attributes) return undefined;
+
+  const headTitle = settings?.attributes.general.headTitle;
+
+  const {
+    socialTitle,
+    titlePlaintext,
+    title,
+    name,
+    number
+  } = entity.attributes;
+  const baseTitle = titlePlaintext ?? title ?? name;
+
+  const entityTitle = maybePrepend(
+    maybeSocialTitle(socialTitle, baseTitle, number, entity.type, t),
+    prepend
+  );
+
+  return appendHeadTitle(
+    maybeAppendParentTitle(entityTitle, parent),
+    headTitle
+  );
+};
+
+export const useMetaDescription = (entity, parent) => {
+  const { socialDescription, descriptionPlaintext } = entity?.attributes ?? {};
+  if (socialDescription) return socialDescription;
+  if (descriptionPlaintext) return descriptionPlaintext;
+
+  const {
+    socialDescription: parentSocialDescription,
+    descriptionPlaintext: parentDescriptionPlaintext
+  } = parent?.attributes ?? {};
+  return parentSocialDescription ?? parentDescriptionPlaintext;
+};
+
+const maybeCoverImage = (entity, fallback) => {
+  if (entity.attributes.coverStyles)
+    return entity.attributes.coverStyles.mediumSquare;
+  return fallback;
+};
```

Comment:
I'd expect this to often be a book or journal cover, in which case we probably don't want to crop it. Let's switch to the `medium` cover style instead.